src/main/java/graphql/execution/Execution.java

```diff
@@ -223,6 +224,10 @@ private CompletableFuture<ExecutionResult> incrementalSupport(ExecutionContext e
     }
 
     private DataLoaderDispatchStrategy createDataLoaderDispatchStrategy(ExecutionContext executionContext, ExecutionStrategy executionStrategy) {
+        if (executionContext.getGraphQLContext().get(DataLoaderDispatchStrategy.CUSTOM_STRATEGY_KEY) != null) {
+            Function<ExecutionContext, DataLoaderDispatchStrategy> customStrategy = executionContext.getGraphQLContext().get(DataLoaderDispatchStrategy.CUSTOM_STRATEGY_KEY);
+            return customStrategy.apply(executionContext);
+        }
```

Comment:
We have

```public @interface ExperimentalApi {
    /**
     * The key that should be associated with a boolean value which indicates whether @defer and @stream behaviour is enabled for this execution.
     */
    String ENABLE_INCREMENTAL_SUPPORT  = "ENABLE_INCREMENTAL_SUPPORT";
}
```

I think we should continue using that place for experimental stuff